src/lib/validator-select_multiple.ts

```diff
@@ -0,0 +1,35 @@
+import {ContactProperty} from '../config';
+import {IValidator} from './validation';
+
+export default class ValidatorSelectMultiple implements IValidator {
+  isValid(input: string, property: ContactProperty): boolean {
+    // Verify property.parameter is an object and is not null
+    if (!property?.parameter || typeof property.parameter !== 'object') {
+      throw new TypeError(`Expected property "parameter" to be an object.`);
+    }
+    const validValues = Object.keys(property.parameter);
+    let selectedValues;
+    if (Array.isArray(input)) {
+      selectedValues = input;
+    } else {
+      selectedValues = input.split(' ').map(value => value.trim()).filter(value => value !== '');
+    }
+    const invalidValues = selectedValues.filter(value => !validValues.includes(value));
+    if (invalidValues.length > 0) {
+      throw new Error(`Invalid values: ${invalidValues.join(', ')}`);
```

Comment:
Should we output the allowed values? I know for your scenario there are a lot... so maybe not?